test/e2e/conftest.py

```diff
@@ -32,6 +72,12 @@ def pytest_configure(config):
     config.addinivalue_line(
         "markers", "slow: mark test as slow to run"
     )
+    config.addinivalue_line(
+        "markers", "function_overrides: function parameters to override when creating fixture"
```

Comment:
PyTest markers are used to label a test - not to provide a parameter to tests. For instance, we have a marker for "slow" tests, in case you want to skip them. I would prefer if we just hardcode the test values, or use another mechanism for [parameterising these ](https://docs.pytest.org/en/6.2.x/parametrize.html)